.travis.yml

```diff
@@ -135,3 +135,24 @@ jobs:
         RUN_FUNCTIONAL_TESTS=false
         GOAL="all deploy"
         BITCOIN_CONFIG="--enable-gui --enable-reduce-exports --enable-werror"
+# OSX bash compatibility
+    - stage: compat
+      os: osx
+      # Use the earliest macOS that can build our lint dependencies:
+      # Xcode 7.3.1, macOS 10.11, JDK 1.8.0_92-b14
+      # Earlier versions failed to make and install ghc for shellcheck
+      # https://docs.travis-ci.com/user/reference/osx/#OS-X-Version
+      osx_image: xcode7.3
+      # TODO: if you're updating osx_image, try using rvm: to supply the version of ruby
+      # required by homebrew. Could not get this working with xcode7.3 but believe it should
```

Comment:
It says `osx_image: xcode7.3` and at the same time `could not get this working with xcode7.3`